### DISCUSSION TITLE
Allow to set monospace font

### DISCUSSION THREAD

**Discussion by CoelacanthusHex** [Docile-Alligator/Infinity-For-Reddit#873] [Ideas]
Allow to set monospace font used to display code block.

**Comment by Wladefant** [comment]
Hello @CoelacanthusHex 
Thank you for your suggestion
Do you want to set a monospace font for displaying code.
Do you have issues how the code is displayed right now?
Could you send some Screenshots?
![SmartSelect_20220712-205102_Infinity.jpg](https://user-images.githubusercontent.com/79611093/178571839-0d599ae4-946a-4ad3-b3a9-9e12bb374724.jpg)
Because I can not see a problem there

**Comment by CoelacanthusHex** [comment]
> Hello @CoelacanthusHex Thank you for your suggestion Do you want to set a monospace font for displaying code. Do you have issues how the code is displayed right now? Could you send some Screenshots? ![SmartSelect_20220712-205102_Infinity.jpg](https://user-images.githubusercontent.com/79611093/178571839-0d599ae4-946a-4ad3-b3a9-9e12bb374724.jpg) Because I can not see a problem there

There's nothing wrong with the display, it's just that I want to use a font that I like, as we can do now for the UI and title fonts.

**Comment by Wladefant** [comment]
> > Hello @CoelacanthusHex Thank you for your suggestion Do you want to set a monospace font for displaying code. Do you have issues how the code is displayed right now? Could you send some Screenshots? ![SmartSelect_20220712-205102_Infinity.jpg](https://user-images.githubusercontent.com/79611093/178571839-0d599ae4-946a-4ad3-b3a9-9e12bb374724.jpg) Because I can not see a problem there
> 
> There's nothing wrong with the display, it's just that I want to use a font that I like, as we can do now for the UI and title fonts.

All right 😄
